src/main/java/core/basesyntax/BinaryString.java

```diff
@@ -7,6 +7,16 @@
      * а возвращает String с представлением этого числа в двоичном виде.
      */
     public String toBinaryString(int value) {
-        return null;
+        if (value == 0) {
+            return "0";
+        }
+        String result = "";
```

Comment:
better use String builder in this solution)

Reply:
Okay, I`ve changed it.